course_catalog/settings.py

```diff
@@ -385,7 +392,5 @@ def get_all_config_keys():
 # List of mandatory settings. If any of these is not set, the app will not start
 # and will raise an ImproperlyConfigured exception
 MANDATORY_SETTINGS = [
```

Comment:
Keep as is, you can enter fake values here for these in your `.env` file for now